wsgi/odyssy/announcement/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [ # pylint: disable=invalid-name
```

Comment:
This is because it is Codacy is flagged error for urlpatterms. That's why i had used this.